src/test/scala/org/broadinstitute/dsde/workbench/sam/azure/AzureServiceSpec.scala

```diff
@@ -0,0 +1,116 @@
+package org.broadinstitute.dsde.workbench.sam.azure
+
+import cats.effect.IO
+import org.broadinstitute.dsde.workbench.sam.ConnectedTest
+import org.broadinstitute.dsde.workbench.sam.Generator.genWorkbenchUserAzure
+import org.broadinstitute.dsde.workbench.sam.TestSupport._
+import org.broadinstitute.dsde.workbench.sam.dataAccess.{MockRegistrationDAO, PostgresDirectoryDAO}
+import org.broadinstitute.dsde.workbench.sam.model.{ResourceId, UserStatus, UserStatusDetails}
+import org.broadinstitute.dsde.workbench.sam.service.{NoExtensions, TosService, UserService}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import scala.jdk.CollectionConverters._
+
+class AzureServiceSpec extends AnyFlatSpec with Matchers with ScalaFutures {
+  implicit val ec = scala.concurrent.ExecutionContext.global
+  implicit val ioRuntime = cats.effect.unsafe.IORuntime.global
+
+  "AzureService" should "create a pet managed identity" taggedAs ConnectedTest in {
+    val azureServicesConfig = appConfig.azureServicesConfig
+
+    assume(azureServicesConfig.isDefined, "-- skipping Azure test")
```

Comment:
Note these tests only run if `render-test-config.sh` was run. This is not set up in CI/CD, so these tests are skipped. Part of https://broadworkbench.atlassian.net/browse/ID-96 should be enabling these tests in CI/CD.